specification/gp-connect-user-permissions.yaml

```diff
@@ -203,6 +204,10 @@ components:
             in non-FHIR APIs for consistency across the platform.
           enum:
             - OperationOutcome
+        id:
```

Comment:
As `id` is for an apigee error I think it should be removed from the spec - the signing service doesn't include it either.